server/src/LspServer.ts

```diff
@@ -60,6 +60,7 @@ export class LspServer {
   dbtRpcServer = new DbtRpcServer();
   dbtRpcClient = new DbtRpcClient();
   openedDocuments = new Map<string, DbtTextDocument>();
+  definitionTriggeredDocuments = new Map<string, number>();
```

Comment:
Maybe we can use just boolean? I think if we trigger definition it can be only on the current document.
`definitionTriggered = false;`

Reply:
Not relevant. See new implementation.